### ISSUE TITLE
desktop-toggle not works

### ISSUE THREAD

**Issue by fasheng** [fasheng/arch-deepin#115]
Run `/usr/lib/deepin-daemon/desktop-toggle` and got errors
```
desktop-toggle[11044]: .:0: Get showing desktop state failed: GetProperty: No such property '_NET_SHOWING_DESKTOP' on window 29f.
```

**Comment by felixonmars** [comment]
Still present in deepin-daemon 2.93.1.20151208-2

**Comment by felixonmars** [comment]
Fix landed with deepin-daemon-2.93.2.20151210-1.
